app-core.js

```diff
@@ -55,20 +62,21 @@ function setupExpressServer() {
 async function ensureSystemSettingsInitiated() {
   logger.info("Checking Server Settings...");
 
-  await ServerSettingsDB.find({}).catch((e) => {
-    if (e.message.includes("command find requires authentication")) {
-      throw "Database authentication failed.";
-    } else {
-      throw "Database connection failed.";
-    }
-  });
-
   // Setup Settings as connection is established
-  const serverSettingsStatus = await ServerSettings.init();
-  await ClientSettings.init();
-  logger.info(serverSettingsStatus);
+  const serverSettingsInitialisation = await initializeServerSettingsCache();
```

Comment:
The lines of code in this function do not add a lot. I'd move the logger to within the cache initialization.